mock/mock.py

```diff
@@ -2059,9 +2059,15 @@ def _format_call_signature(name, args, kwargs):
     message = '%s(%%s)' % name
     formatted_args = ''
     args_string = ', '.join([repr(arg) for arg in args])
-    kwargs_string = ', '.join([
-        '%s=%r' % (key, value) for key, value in sorted(kwargs.items())
-    ])
+
+    kwargs_to_join = []
+    for key, value in sorted(kwargs.items()):
+        if six.PY2 and isinstance(key, unicode):
```

Comment:
We can make this a simple utility function to avoid unnecessary churn:

```py
   kwargs_string = ', '.join([
        '%s=%r' % (encode_item(key), value) for key, value in sorted(kwargs.items())
    ])
```